tests/10apidoc/01register.pl

```diff
@@ -61,6 +61,32 @@
       });
    };
 
+test "POST /createUser can get or create a user",
+   requires => [ $main::API_CLIENTS[0] ],
+
+   critical => 1,
```

Comment:
Yes it doesn't sound like this is a critical test.

A `critical` test is one that is deemed so important to the entire remaining test suite, that if it fails we should just bail out and stop here. That's marked on things like the regular user registration test, because if sytest can't register users at all, there's very little point continuing at all.